tests/affiliatedcertification/tests/affiliated_certification_container_is_certified_digest.go

```diff
@@ -126,13 +126,13 @@ var _ = Describe("Affiliated-certification container-is-certified-digest,", func
 		dep := deployment.DefineDeployment("affiliated-cert-deployment", randomNamespace,
 			tsparams.UncertifiedContainerURLCnfTest, tsparams.TestDeploymentLabels)
 
-		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.Timeout)
+		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(globalhelper.GetAPIClient().K8sClient.AppsV1(), dep, tsparams.Timeout)
```

Comment:
Would it make sense to create a new helper function instead and make globalhelper.CreateAndWaitUntilDeploymentIsReady a wrapper function just using obalhelper.GetAPIClient().K8sClient.AppsV1(). Also this variable is local to globalhelper. Another bonus is that there would be much less changes/repetitions

Reply:
That's a good idea.  I'll do that!

Reply:
Done.